agent/engine/dependencygraph/graph_test.go

```diff
@@ -476,6 +479,14 @@ func TestVerifyResourceDependenciesResolved(t *testing.T) {
 			RequiredStatus:   resourcestatus.ResourceStatus(1),
 			ExpectedResolved: true,
 		},
+		{
+			Name:             "resource none,container pull depends on resource created",
```

Comment:
isn't this UT duplicated?

Reply:
Oh sorry it is.. I did a refactor and removed the changes that need this new test case but forgot to remove the test.